src/guardian/staking-module-guard/staking-module-guard.service.ts

```diff
@@ -412,6 +425,30 @@ export class StakingModuleGuardService {
     await this.guardianMessageService.sendDepositMessage(depositMessage);
   }
 
+  public async getInvalidKeys(
+    stakingModuleData: StakingModuleData,
+    blockData: BlockData,
+  ): Promise<{ key: string; depositSignature: string }[]> {
+    this.logger.log('Start keys validation', {
+      keysCount: stakingModuleData.vettedKeys.length,
+    });
+    const validationTimeStart = performance.now();
+    const invalidKeysList = await this.keysValidationService.validateKeys(
```

Comment:
Do we need to validated vetted used keys?

Reply:
no

Reply:
and we even dont fetch vetted used keys

Reply:
vettedKeys -> vettedUnusedKeys